.vscode/settings.json

```diff
@@ -5,5 +5,6 @@
     "esbonio.sphinx.confDir": "${workspaceFolder}/docs",
     "restructuredtext.pythonRecommendation.disabled": true,
     "restructuredtext.preview.name": "sphinx",
-    "gitlens.showWhatsNewAfterUpgrades": false
+    "gitlens.showWhatsNewAfterUpgrades": false,
+    "restructuredtext.linter.rstcheck.executablePath": "rstcheck"
```

Comment:
Fixes the rstcheck vscode integration. Source: https://github.com/vscode-restructuredtext/vscode-restructuredtext/issues/400#issuecomment-1193186039

However, the maintainers mention that rstcheck shouldn't be necessary anymore as Esbonio reports most of the issues too: https://github.com/vscode-restructuredtext/vscode-restructuredtext/issues/400#issuecomment-1193188900. I'm happy to remove it altogether but Esbonio certainly doesn't catch everything yet.